### ISSUE TITLE
Array Sync issue

### ISSUE THREAD

**Issue by kvnh** [3lvis/Sync#565]
I have a Binary Data attribute 'templateRequirementManager' set up in a Core Data entity but when I try to sync the following JSON to the entity it does not sync.
{
    "template_requirement_manager": [1, 2, 3]
}

I have ran a simple test replacing the camel case templateRequirementManager with manager and syncing the following JSON which in fact works and a binary blob is saved to the DB.
{
    "trmanager": [1, 2, 3]
}

Is there something additional I need to do for camel case Binary Data attributes to get them to work?

**Comment by kvnh** [comment]
I have managed to fix the issue. Not sure if its a problem with Core Data or Sync - more likely Core Data. 
The 'templateRequirementManager' attribute had previously been created with a String type - updating it to a Binary Data type was when the problems began. Creating a brand new attribute 'trmanager' of type Binary Data worked because it did not have a previous type. 
Is this because Core Data caches values? I deleted the app and Derived Data multiple times but the 'templateRequirementManager' attribute still didn't sync down the expected array so I deleted it and created a brand new attribute with the same name and it is now working as intended. Happy to close out if you think the issue is related to Core Data.
